pallets/moonbeam-orbiters/src/lib.rs

```diff
@@ -124,6 +128,12 @@ pub mod pallet {
 	/// Current round index
 	pub(crate) type CurrentRound<T: Config> = StorageValue<_, T::RoundIndex, ValueQuery>;
 
+	#[pallet::storage]
+	/// If true, it forces the rotation at the next round.
+	/// A use case: when changing RotatePeriod, you need a migration code that sets this value to
+	/// true to avoid holes in OrbiterPerRound.
+	pub(crate) type ForceRotation<T: Config> = StorageValue<_, bool, ValueQuery>;
```

Comment:
Do we foresee situations in which we will need to set this value through a root-callable extrinsic? It might be interested to add it just in case there are edge cases we dont foresee